### PR TITLE
Add guidelines to contributing in Community Handbook

### LINKED ISSUE #4513: [Handbook] Contribution Guide Update w/Issue etiquette
Please summarize the following information as an update with the [Community Handbook](https://layer5.io/community/handbook).

---

              > Guys please don't create PRs until an issue has been assigned to you.

Sound counsel. Don’t get us wrong, we certainly appreciate the enthusiasm, however, confirming assignment, first will avoid heartache in the form of:

1. Issue not qualified: effort put into an invalid issue.
2. Contributor conflict: a bit of stepping on toes and not so kindly pushing others out of the way. Certainly don't want the project to move slowly, but a little bit of kindness goes a long way.
3. Maintainer cycles spent clarifying and assuaging ruffled feathers (hurt feelings).

Regarding where to go from here:
- Is the issue described above valid? _Yes_
- Do we want this issue fixed? _Yes_
- Do we generally assign issues in chronological order of contributor request? _Yes_
- Are there exceptions to that rule? _Yes_
- If yes, when? _When the requesting contributor is 1) unqualified, or 2) unavailable or incapable of delivering in a timely fashion, or 3) missing too much context and whereas another contributor already has that context, or 4) when a requesting contributor already has one or more other open assignments._

@Saksham294, will you do me a favor and place this info into an FAQ and/or section of the Community Handbook?

@RakshitKumar04, since you have an open PR and the issue is valid, we'll move forth with your assignment. Take note of @Saksham294's counsel for future reference, if you would, please. My counsel is to take a moment to soften your phrasing; too apply a touch of kindness in your interactions.

_Originally posted by @leecalcote in https://github.com/layer5io/layer5/issues/4492#issuecomment-1627561901_

### PR DESCRIPTION
**Pull request by Saksham294**
**Description**
The current community handbook lacked the guidelines for new contributors in making their first and subsequent contributions. This PR adds new guidelines about what and what not to do while contributing and interacting with other community members.

This PR fixes #4513 

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 2813a851033a08d75ad4d3631e5f3d685f970d19 at: https://64aaffe33571ea529b653041--layer5.netlify.app

**Comment by sudhanshutech** [comment]
@Saksham294 Let's discuss this on Websites Meeting call on Monday. Please add this as an agenda item in the [meeting mintes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#heading=h.69qskjv558bk) if you would

File: src/sections/Community/Handbook/faq.js [hunk 1]
```diff
@@ -41,12 +41,35 @@ const Faq = () => {
...
+              <li>Minimize Maintainer Burden: Adhere to guidelines to reduce maintainers' time spent on clarifications and resolving conflicts, enabling them to focus on essential tasks.</li>
+            </ul>
+            <h4>Q: What should I consider when assigning issues to contributors?</h4>
+            <p>A: When assigning issues, follow these principles:</p>
+            <ul>
+              <li>Valid Issue: Assign issues that require attention.</li>
+              <li>Fix the Issue: Prioritize resolving them promptly.</li>
+              <li>Assignment Order: Generally chronological, with exceptions:</li>
+              <ul>
+                <li>Unqualified, unavailable, or incapable contributors.</li>
```
Comment (thisiskaransgit) [inline src/sections/Community/Handbook/faq.js:58]: @Saksham294 Let's break the first line
- Unqualified to perform this task currently.
- Unavailable and/or incapable of performing the task promptly.

File: src/sections/Community/Handbook/faq.js [hunk 2]
```diff
@@ -41,12 +41,35 @@ const Faq = () => {
...
+                <li>Lack of context when another contributor has it.</li>
+                <li>Requesting contributor with existing open assignments.</li>
```
Comment (thisiskaransgit) [inline src/sections/Community/Handbook/faq.js:60]:
```suggestion
                <li>Requesting contributor has existing open assignments.</li>
```

File: src/sections/Community/Handbook/faq.js (hunk 1 above)
Comment (Saksham294) [reply src/sections/Community/Handbook/faq.js]: Done @thisiskaransgit

File: src/sections/Community/Handbook/faq.js (hunk 2 above)
Comment (Saksham294) [reply src/sections/Community/Handbook/faq.js]: Done @thisiskaransgit

**Comment by l5io** [comment]
🚀 Preview for commit 72366355cf68e6e0decd0f1275c775960f0fd98b at: https://64abe866ad158c0529fd443a--layer5.netlify.app

**Review by thisiskaransgit — APPROVED** [review]
Looks good to merge, thank you for your contribution :tada:

**Review by hirentimbadiya — APPROVED** [review]
LGTM ✅

**Review by Nikhil-Ladha — APPROVED** [review]
LGTM

**Pull request merged**